edx_rest_framework_extensions/auth/jwt/middleware.py

```diff
@@ -208,17 +211,37 @@ def process_view(self, request, view_func, view_args, view_kwargs):  # pylint: d
         """
         assert hasattr(request, 'session'), "The Django authentication middleware requires session middleware to be installed. Edit your MIDDLEWARE setting to insert 'django.contrib.sessions.middleware.SessionMiddleware'."  # noqa E501 line too long
 
+        # # TODO: Move pseudocode to helper function and make this work.
+        # if get_setting(ENABLE_FORGIVING_JWT_COOKIES):
+        #     # JSONWebTokenAuthentication gives precedence to a JWT in the authorization header, before
+        #     # using the JWT Cookie. Don't reconstitute the JWT Cookie unless it would be used.
+        #     # This simplifies later code that wants to rely on the JWT Cookie and isn't aware of
+        #     # this order of precendence.
+        #     # See https://github.com/Styria-Digital/django-rest-framework-jwt/blob/befefaa1ac1a57b1f207639a7ead472094093642/src/rest_framework_jwt/authentication.py#L90-L98
+        #     reconstitute_jwt_cookie = False
+        #     try:
+        #         authorization_header = force_str(JwtAuthentication.get_authorization_header(request))
+        #         JwtAuthentication.get_token_from_authorization_header(authorization_header)
+        #     except InvalidAuthorizationCredentials:
+        #         reconstitute_jwt_cookie = True
+        # else:
+        #     reconstitute_jwt_cookie = request.META.get(USE_JWT_COOKIE_HEADER)
+        reconstitute_jwt_cookie = request.META.get(USE_JWT_COOKIE_HEADER)
```

Comment:
**Reviewer Note:** This was a temporary line just to get more unit tests to pass temporarily.